CHANGELOG.md

```diff
@@ -81,6 +81,9 @@ at locations that better optimize for object storage.
 
 - Introduced bootstrap command options to specify custom schema files for database initialization.
 
+- Added refresh credentials endpoint configuration to LoadTableResponse for AWS, Azure, and GCP. Enabling
+automatic storage credential refresh per table on the client side.
```

Comment:
Lets call out the client version limitation here and the suggestion to the caller to set the client side override to enable this.

Reply:
good points!

Reply:
since the release is imminent, if the round trip cycle is slow, I suppose we can still merge this PR and later adjust the changelog between @singhpk234 and myself :) Let's give it a few more hours.